symExec.py

```diff
@@ -734,8 +734,9 @@ def sym_exec_ins(start, instr, stack, mem, global_state, path_conditions_and_var
             raise ValueError('STACK underflow')
     elif instr_parts[0] == "MOD":
         if len(stack) > 1:
-            first = stack.pop(0)
-            second = stack.pop(0)
+            first = unsigned_format(stack.pop(0))
```

Comment:
@ngotungson check if `first` is symbolic and handle it similar to [this](https://github.com/melonproject/oyente/blob/a333ea2c74c856407e8369fd9e0e3deed14cd128/symExec.py#L901)